src/library/scala/collection/immutable/Vector.scala

```diff
@@ -953,11 +953,7 @@ private[immutable] trait VectorPointer[T] {
 
     // STUFF BELOW USED BY APPEND / UPDATE
 
-    private[immutable] final def copyOf(a: Array[AnyRef]) = {
-      val b = new Array[AnyRef](a.length)
-      Platform.arraycopy(a, 0, b, 0, a.length)
-      b
-    }
+    private[immutable] final def copyOf(a: Array[AnyRef]) = java.util.Arrays.copyOf(a, a.length)
```

Comment:
@retronym - On my machine, the copyOf version is actually better-optimized than the new Array/arraycopy.  I don't see anything left in the assembly of the object type test, and the it looks a lot like the 2.11.8 version.